packages/translations/src/i18next/i18next.ts

```diff
@@ -134,10 +134,10 @@ export const changeLanguage = async (lang, cb) => {
 // <Localize /> component wrapped with i18n
 export const Localize = withI18n(i18n);
 
-export const localize = (string, values) => {
+export const localize = (string: string, values?: object) => {
```

Comment:
```suggestion
export const localize = <T extends object>(string: string, values?: T) => {
```

what do you think of using generic type for this `value` instead of `object`?

Reply:
Sure @niloo-fs , Let me check and update this. 🙏🏽